### PR TITLE
Add cherrypy to compatibility table, update ja/fr tables

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### What does this PR do?

Adds cherrypy to the compatibility table for the python client. Also updates the web framework table for the fr/ja docs.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
Hi @Kyle-Verhoog! You only need to update the english files, our translation pipelines take care of the rest for japanese and french! Can you please update to just english file and rebase so there's no conflicts?

Thanks!

**Review by jtappa — CHANGES_REQUESTED** [review]
Just needs english only files updated and a rebase

**Comment by Kyle-Verhoog** [comment]
:+1: should be good to go now @jtappa 🙂

File: content/en/tracing/setup_overview/compatibility_requirements/python.md [hunk 1]
```diff
@@ -30,18 +30,19 @@ The `ddtrace` library includes support for a number of web frameworks, including
...
-| [FastAPI][9]              | >= 0.51           | https://ddtrace.readthedocs.io/en/stable/integrations.html#fastapi |
-| [Molten][10]               | >= 0.7.0          | https://ddtrace.readthedocs.io/en/stable/integrations.html#molten  |
-| [Pylons][11]              | >= 0.9.6          | https://ddtrace.readthedocs.io/en/stable/integrations.html#pylons  |
-| [Pyramid][12]             | >= 1.7            | https://ddtrace.readthedocs.io/en/stable/integrations.html#pyramid |
-| [pytest][13]              | >= 3.0            | https://ddtrace.readthedocs.io/en/stable/integrations.html#pytest  |
-| [Sanic][14]               | >= 19.6.0         | https://ddtrace.readthedocs.io/en/stable/integrations.html#sanic   |
-| [Starlette][15]           | >= 0.13.0         | https://ddtrace.readthedocs.io/en/stable/integrations.html#starlette |
-| [Tornado][16]             | >= 4.0            | https://ddtrace.readthedocs.io/en/stable/integrations.html#tornado |
+| [CherryPy][6]            | >= 11.2.0         | https://ddtrace.readthedocs.io/en/stable/integrations.html#cherrypy|
```
Comment (jtappa) [inline content/en/tracing/setup_overview/compatibility_requirements/python.md:33]: Looks like link 6 at the bottom of the file is still Django, which means all the links are now off by one.

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**